projects/plugins/jetpack/modules/related-posts/jetpack-related-posts.php

```diff
@@ -85,6 +85,15 @@ protected function get_blog_id() {
 		return Jetpack_Options::get_option( 'id' );
 	}
 
+	/**
+	 * Determine if the site is running an FSE theme.
+	 *
+	 * @return bool True if the current theme is an FSE/Site Editor theme
+	 */
+	protected function is_fse_theme() {
+		return function_exists( 'gutenberg_is_fse_theme' ) && gutenberg_is_fse_theme();
```

Comment:
I agree! Do you think it would be reasonable to extract this in a follow-up PR? Happy to include it here if preferred, I'll just need to sort out the testing.